libs/perun/dialogs/src/lib/applications-bulk-operation-dialog/applications-bulk-operation-dialog.component.html

```diff
@@ -0,0 +1,52 @@
+<h1 mat-dialog-title>{{ title | translate }}</h1>
+
+<p mat-dialog-content>
+  <perun-web-apps-alert alert_type="warn">
+    {{ description | translate: {count: dataSource.data.length} }}
+  </perun-web-apps-alert>
+</p>
+
+<div mat-dialog-content class="dialog-content mx-auto">
+  <mat-table [dataSource]="dataSource" matSort>
+    <ng-container *ngFor="let column of displayedColumns" [matColumnDef]="column">
+      <mat-header-cell *matHeaderCellDef mat-sort-header> {{column}} </mat-header-cell>
+      <mat-cell *matCellDef="let element"> {{element[column]}} </mat-cell>
+    </ng-container>
+    <mat-header-row *matHeaderRowDef="displayedColumns; sticky: true"></mat-header-row>
+    <mat-row *matRowDef="let row; columns: displayedColumns;"></mat-row>
+  </mat-table>
+</div>
+
+<div mat-dialog-actions class="ms-auto justify-content-end">
+  <button class="me-2" mat-stroked-button (click)="onCancel()">
+    {{'VO_DETAIL.APPLICATION.DIALOG.CANCEL' | translate}}
+  </button>
+  <button
+    *ngIf="action === 'APPROVE'"
+    color="accent"
+    class="me-2"
+    mat-flat-button
+    (click)="onConfirm()">
+    {{'VO_DETAIL.APPLICATION.APPLICATION_DETAIL.APPROVE' | translate}}
+  </button>
```

Comment:
If at the end confirm will just close dialog it would be nicer to have one button just set the proper label beforehand.

Reply:
I made them 4 separate buttons since 'RESEND' is mat-stroked-button, unlike the other 3 that are mat-flat-button. And within those 3 there's also difference in color of 'APPROVE' button. So it is not just about the label.

Reply:
I see but in this case, as a confirmation dialog, it might be more in line with other dialogs to use flat accent button with label of the action for all cases.